trunk/examples/witness-checking/regression/sequential/violation/assumption.c

```diff
@@ -0,0 +1,4 @@
+int main(){
+  int x = __VERIFIER_nondet_int();
+  if (x >= 0) reach_error();
+}
```

Comment:
> Could you add an expected verdict (i.e. `//#False`) ?

What do you mean with this verdict? Currently all witness regression tests have an implicit verdict based on their folder, i.e., we assume that all witnesses are valid.

> I guess it works for us even without declarations of these functions. Would be nicer though if they were present.

Yes, I can change that.

Reply:
I meant a verdict for program correctness, which, in this folder, would always be `False`. Just in case these programs ever occur in any other context.

Though you raise an interesting point that I also meant to raise in our discussions of correctness witnesses: We should also have tests for invalid witnesses to see they are indeed rejected. Though I wouldn't write this verdict in the program (there may be multiple witnesses for a program, some valid and some invalid).